src/utils/pureComments.ts

```diff
@@ -3,6 +3,14 @@ import * as acorn from 'acorn';
 import { base as basicWalker } from 'acorn-walk';
 import { CommentDescription } from '../Module';
 
+// patch up acorn-walk until class-fields are officially supported
+basicWalker.FieldDefinition = function (node: any, st: any, c: any) {
+	c(node.key, st, 'Expression');
```

Comment:
Shouldn't this line to be `if ( node.computed ) { c(node.key, st, 'Expression'); }`, just like `MethodDefinition` / `Property` do in `acorn-work`?

Reply:
Good catch, was obviously copying without thinking.